code/modules/antagonists/abductor/equipment/gear/abductor_posters.dm

```diff
@@ -17,6 +17,12 @@
 		return
 	return ..()
 
+/obj/structure/sign/poster/abductor/attackby(obj/item/tool, mob/user, params)
+	if(tool.toolspeed >= 0.2)
+		to_chat(user, span_warning("Your [tool.name] is too weak to cut through the superior adhesive on the [name]!"))
```

Comment:
```suggestion
		balloon_alert(user, "tool too weak!")
```
This doesn't really seem important enough to need to stick around in the chat log.